composer.json

```diff
@@ -18,7 +18,8 @@
         "chumper/zipper": "^0.6.1",
         "guzzlehttp/guzzle": "^6.2",
         "predis/predis": "^1.1",
-        "rairlie/laravel-locking-session": "^1.1"
+        "rairlie/laravel-locking-session": "^1.1",
+        "doctrine/dbal": "2.5.13"
```

Comment:
I'm not sure what the effect of the addition of `"doctrine/dbal": "2.5.13"` here is.

Reply:
This is requires to run the Laravel  migrate functionality on the sqlite database. 

Although we are not doing that in this refactor (it will be done in the move to 5.8) but I discovered (while switching from one branch to another in bfumatest) that it was missing. The composer.lock file actually contained the following suggestion: 

`"suggest": {
...
    "doctrine/dbal": "Required to rename columns and drop SQLite columns (~2.4).",`

So in the end I decided to include this advice in the refactor because it makes your life easier when switching branches if that involves database changes. (When we move away from sqlite this will not be needed)